### PR TITLE
Add es to available locales in US config

### PR DESCRIPTION
**Pull request by andrewpbrett**
Pretty self-explanatory? :)

### CODE REVIEW

File: inventory/group_vars/us.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ checkout_zone: USA
 country_code: US
 currency: USD
 locale: en_US
-available_locales: en_US
+available_locales: en_US, es
```
Comment (sauloperez) [inline inventory/group_vars/us.yml:7]: I keep that `es` but in the future, we may want to use `es_US` instead.

**Review by sauloperez — APPROVED** [review]
Indeed, self-explanatory :joy:

**Comment by andrewpbrett** [comment]
Yes! Laurie actually corrected me on that, I misunderstood something she said. Just pushed a fix.

On Sun, Apr 4, 2021, at 00:31, Pau Pérez Fabregat wrote:
> 

> ***@***.**** approved this pull request.

> Indeed, self-explanatory 😂

> 
> In inventory/group_vars/us.yml <https://github.com/openfoodfoundation/ofn-install/pull/723#discussion_r606761354>:

> > @@ -4,7 +4,7 @@ checkout_zone: USA
 country_code: US
 currency: USD
 locale: en_US
-available_locales: en_US
+available_locales: en_US, es

File: inventory/group_vars/us.yml (hunk 1 above)
Comment (RachL) [reply inventory/group_vars/us.yml]: @andrewpbrett @sauloperez I would do right away an es_US : remember that the main menu is made with translations. So the about link in spanish could never go to the USwebsitee and would always link to Katuma's.
We have this in FR right now with the IT translation and it's pretty annoying.

**Comment by lauriewayne** [comment]
Oh I can see where the "about" menu item would for sure be a pain! Hope we'll have a Spanish reviewer soon who can keep up with new terms and it will make sense for us to use es_US. Thank you all! <3

**Review by Matt-Yorkley — APPROVED** [review]

**Pull request merged**